### PR TITLE
Query parameter for timeline view

### PR DESCRIPTION
**Pull request by mvidalgarcia**

### CODE REVIEW

File: indico/modules/rb_new/client/js/components/pages/BookRoom.jsx [hunk 1]
```diff
@@ -64,12 +64,7 @@ export default class BookRoom extends React.Component {
         suggestions: PropTypes.object.isRequired
     };
 
-    constructor(props) {
-        super(props);
-        this.state = {
-            view: 'book'
-        };
-    }
+    state = {};
```
Comment (ThiefMaster) [inline indico/modules/rb_new/client/js/components/pages/BookRoom.jsx]: can be omitted, especially if there's no local state

File: indico/modules/rb_new/client/js/components/pages/BookRoom.jsx [hunk 2]
```diff
@@ -284,12 +277,17 @@ export default class BookRoom extends React.Component {
         );
     }
 
+    switchToRoomList = () => {
+        const {setFilterParameter} = this.props;
+        setFilterParameter('showTimeline', false);
```
Comment (ThiefMaster) [inline indico/modules/rb_new/client/js/components/pages/BookRoom.jsx]: it's not really a "filter parameter"...

File: indico/modules/rb_new/client/js/components/pages/BookRoom.jsx (hunk 1 above)
Comment (mvidalgarcia) [reply indico/modules/rb_new/client/js/components/pages/BookRoom.jsx]: There is local state.

**Comment by mvidalgarcia** [comment]
Updated and adapted to the changes on https://github.com/indico/redux-router-querystring/pull/1

**Comment by mvidalgarcia** [comment]
Rebased

**Pull request merged**